### PR TITLE
GHG updates

### PR DESCRIPTION
**Pull request by danielfdsilva**
Closes https://github.com/NASA-IMPACT/veda-architecture/issues/276

Currently veda-ui is pointing to https://github.com/NASA-IMPACT/veda-ui/pull/553 given that unmerged features are needed.

## Please note

The changes here are directly taken from the suggested placeholder content and might need more thorough design review before considered a proper suggestion for an official site.

### CODE REVIEW

**Comment by danielfdsilva** [comment]
New URL: https://deploy-preview-7--ghg-demo.netlify.app/

File: overrides/home.mdx [hunk 1]
```diff
@@ -30,4 +42,44 @@ import FeaturedDiscoveries from "$veda-ui-scripts/components/home/featured-disco
   </Prose>
 </Block>
 
+<Block>
```
Comment (j08lue) [inline overrides/home.mdx:45]: Logos should be in alphabetical order, please.

File: overrides/partnership.mdx [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
+menu: Partnership
+title: Our Partners
+description: 
+---
+
+<Block>
```
Comment (j08lue) [inline overrides/partnership.mdx:7]: Partners should be listed in alphabetical order, please.

**Review by j08lue — APPROVED** [review]
This nicely implements the changes requested with the content provided.

This is ready for merging. Any updates, such as changing the order of the partners mentioned, can be done in a new PR.

Thanks for the quick action, @danielfdsilva.

File: overrides/home.mdx [hunk 3]
```diff
@@ -30,4 +42,48 @@ import FeaturedDiscoveries from "$veda-ui-scripts/components/home/featured-disco
...
+  <Prose>
+    ## Our partners
+
+    <LogoWrapper>
+      <a href="https://www.epa.gov/">
+        <Image
+          src={new URL("./media/epa.svg", import.meta.url).href}
+          alt="epa logo"
+          width="100"
+          height="100"
```
Comment (danielfdsilva) [inline overrides/home.mdx:55]: @siddharth0248 It is better if the images don't have a height as they're sized proportionally.

**Review by danielfdsilva — COMMENTED** [review]
@siddharth0248 Additionally, ping me once this is ready so I can make a final change before merging

**Pull request merged**